apps/artemis-web-e2e/src/integration/test_api.spec.ts

```diff
@@ -69,4 +69,24 @@ describe('artemis-web-api', () => {
 
         });
     });
+
+    it('[API] login rate limit', () => {
```

Comment:
does this test pass correctly? Is the 429 code also demonstrated in the UI? If you have tested, could you also attach some screenshots here?